src/Sets/Zonotope.jl

```diff
@@ -387,3 +387,55 @@ function split(Z::Zonotope, j::Int)
     Z₂ = Zonotope(c₂, G₂)
     return Z₁, Z₂
 end
+
+"""
+    quadratic_map(Z::Zonotope{N}, Q::Array{A}) where {N, A<:Array{N}}
+
+Return the overapproximation of the quadratic map of the given zonotope.
+
+### Input
+
+- `Z` -- zonotope
+- `Q` -- array of square matrices
+
+### Output
+
+The overapproximation of the quadratic map of the given zonotope.
+
+### Algorithm
+
+This function implements [Lemma 1, 1]
+
+[1] *Matthias Althoff and Bruce H. Krogh. 2012. Avoiding geometric intersection
+operations in reachability analysis of hybrid systems. In Proceedings of the
+15th ACM international conference on Hybrid Systems: Computation and Control
+(HSCC ’12). Association for Computing Machinery, New York, NY, USA, 45–54.*
+"""
+function quadratic_map(Z::Zonotope{N}, Q::Array{A}) where {N, A<:Array{N}}
```

Comment:
I suggest to swap the arguments as in `linear_map`. And also to remove the type restrictions because you should also be able to pass different matrix types.
```suggestion
function quadratic_map(Q, Z::AbstractZonotope{N}) where {N}
```

Reply:
```julia
function quadratic_map(Q::Vector{MT}, Z::AbstractZonotope{N}) where {N, MT<:AbstractMatrix{N}}
...
```